[docs] Rename "Customization > Theme > Global" to "Customization > Theme > Components" (#24115)
mui-org/material-ui: docs/src/pages/customization/theme-components/theme-components.md

# Components

> <p class="description">The `theme.components` key allows you to customize components without using wrappers. You can change the styles, the default props, and more.</p>

Reviewer: ```suggestion
<p class="description">The theme's `components` key allows you to customize a component without wrapping it in another component. You can change the styles, the default props, and more.</p>
```

## Global style overrides

You can use the theme's `styleOverrides` key to potentially change every single style injected by Material-UI into the DOM.

```js
const theme = createMuiTheme({
  components: {
    // Name of the component
    MuiButton: {
      styleOverrides: {
        // Name of the slot
        root: {
          // Some CSS
          fontSize: '1rem',
        },
      },
    },
  },
});
```

{{"demo": "pages/customization/theme-components/GlobalThemeOverride.js"}}

> The list of each component's classes is documented under the **CSS** section of each component's API page.

Reviewer: ```suggestion
The list of each component's classes is documented under the **CSS** section of its API page.
```

});
```

{{"demo": "pages/customization/theme-components/DefaultProps.js"}}

To override lab component styles with TypeScript, check [this page](/components/about-the-lab/#typescript).

## Adding new component variants

> You can use the `variants` key in the theme's `components` section to add new variants to Material-UI components. These new variants can specify what styles the component should have when specific props are applied together.

Reviewer: ```suggestion
You can use the `variants` key in the theme's `components` section to add new variants to Material-UI components. These new variants can specify what styles the component should have when specific props are applied.
```
